src/components/Issuance/Issuance.tsx

```diff
@@ -155,34 +270,102 @@ const IssueCred = () => {
 			setSchemaLoader(false);
 		}
 	};
+	const createW3cSchemaPayload = async (schemaId: string, parsedW3cSchemaDetails: any) => {
+		if (schemaId) {
 
-	const getSelectedUsers = async (): Promise<SelectedUsers[]> => {
-		const selectedUsers = await getFromLocalStorage(storageKeys.SELECTED_USER);
-		return JSON.parse(selectedUsers);
+			if (parsedW3cSchemaDetails) {
+				setW3cSchemaDetails(parsedW3cSchemaDetails);
+				
+				setSchemaLoader(false);
+
+			}
+		}
 	};
 
-	const handleSubmit = async (values: IssuanceFormPayload) => {
-		const issuancePayload = {
-			credentialData: values.credentialData.map((item) => {
-				return {
-					...item,
-					attributes: item.attributes.map((attr) => ({
-						name: attr.name,
-						value: attr.value.toString(),
-					})),
-				};
-			}),
-			credentialDefinitionId: values.credentialDefinitionId,
-			orgId: values.orgId,
-		};
+const getSelectedUsers = async (): Promise<SelectedUsers[]> => {
+		try {
+		  const selectedUsers = await getFromLocalStorage(storageKeys.SELECTED_USER);
+		  if (!selectedUsers) {
+			return [];
+		  }
+		  return JSON.parse(selectedUsers);
+		} catch (error) {
+		  console.error("Error parsing selectedUsers:", error);
+		  return [];
+		}
+	  };
 
+
+	const handleSubmit = async (values: IssuanceFormPayload) => {
+	
+		let issuancePayload;
+	
+		if (!w3cSchema) { 
+			issuancePayload = {
+				credentialData: values.credentialData.map((item) => {				
+					return {
+						...item,
+						attributes: item?.attributes?.map((attr) => ({
+							name: attr.name,
+							value: attr.value.toString(),
+						})),
+					};
+				}),
+				
+				credentialDefinitionId: values.credentialDefinitionId, 
+				orgId: values.orgId,
+			};
+		} 
+		if(w3cSchema) { 
+			issuancePayload = {
+				credentialData: values?.credentialData.map((item) => {
+					return {
+						connectionId: item.connectionId,
+						credential: {
+							"@context": [
+								"https://www.w3.org/2018/credentials/v1",
```

Comment:
We can move these hardcoded values to one common file and use those variables when forming the payload